src/App/Repository/Form/AttributeRepository.php

```diff
@@ -276,6 +271,16 @@ public function getByForms($form_ids, array $include_attributes = null)
 				'form_stage_priority' => 0,
 				'priority' => 1
 			],
+			[
+				'label' => 'Survey',
+				'key' => 'form_name',
+				'type' => 'form_name',
+				'input' => 'text',
+				'form_id' => 0,
+				'form_stage_id' => 0,
+				'form_stage_priority' => 0,
+				'priority' => 2
```

Comment:
I think Priority for all these attributes should be 0 otherwise they will always move around amongst the custom attributes, What do you think?

Reply:
ah. No because we use the stage priority and form id as well to sort. So a stage priority 0 with priority 2 is still lower than custom attributes, and will always appear first